db_tasks.py

```diff
@@ -47,22 +69,52 @@ def databases_are_equal(dburl_a, dburl_b):
         return not m.statements
 
 
-def do_schema_dump(dburl, outfile):
-    COMMAND = 'pg_dump --no-owner --no-privileges --schema-only --column-inserts -f {} {}'
-
-    command = COMMAND.format(outfile, dburl)
-
-    print('MAKING DUMP OF SCHEMA: '.format(command))
-    subprocess.check_output(command, shell=True)
-    print('DUMP COMPLETE')
-
-
-def do_migration(REAL_DB_URL):
+def do_schema_dump(outfile, cld_url, export_url, username, password, service_binding):
+    ctx = ssl.create_default_context()
+
+    print('GETTING TOKEN')
+    conn = httplib.HTTPSConnection(cld_url, 443, context=ctx)
+    conn.request("POST", "/oauth/token", urllib.urlencode({
+        "grant_type": "password",
+        "scope": ",".join(["openid", "cloud_controller.read"]),
+        "username": username,
+        "password": password,
+    }), {
+        "Authorization": "Basic %s" % base64.b64encode("%s:%s" % ("db-export-tool-api", "notasecret")),
+        "Content-Type": "application/x-www-form-urlencoded",
+    })
+    resp = conn.getresponse()
+    if resp.status != 200:
+        raise Exception('Could not get token from {}'.format(cld_url))
+    token = json.loads(resp.read())["access_token"]
+    conn.close()
+
+    print('DOWNLOADING SCHEMA')
+    conn = httplib.HTTPSConnection(export_url, 443, context=ctx)
+    conn.request("GET", "/dbSchema?%s" % urllib.urlencode({
+        "servicebinding": service_binding,
+    }), None, {
+        "Authorization": "Bearer %s" % token,
+    })
+
+    resp = conn.getresponse()
+    if resp.status != 200:
+        raise Exception('Could not get schema from {}'.format(export_url))
+    schema = resp.read()
+    conn.close()
+
+    # strip out all chars before first '-'
+    schema = schema[schema.find('-')::]
```

Comment:
Should be a single `:` not `::` (or just remove this line as we no longer output the header).